ReSharper.FSharp/src/FSharp.Psi.Daemon/src/Stages/FcsErrorsStageProcessBase.fs

```diff
@@ -432,8 +433,9 @@ type FcsErrorsStageProcessBase(fsFile, daemonProcess) =
             | null -> null
             | memberDecl -> InstanceMemberRequiresTargetError(memberDecl) :> _
 
+        | LiteralPatternDoesNotTakeArguments
         | UnionCaseDoesNotTakeArguments ->
-            createHighlightingFromNode UnionCaseDoesNotTakeArgumentsError range
+            createHighlightingFromNode RemovePatternArgumentError range
```

Comment:
This change is wrong. It will now create a highlighting with number 725 for an error with number 3191.

Reply:
So it needs to be a new quick fix with the same logic ?

Reply:
No, one quick fix may work with different errors/warnings, please check the existing quick fixes that have multiple constructors.

Reply:
Done